easypy/signals.py

```diff
@@ -215,7 +235,41 @@ def registered(self, func, **kwargs):
         finally:
             self.unregister(func)
 
+    def _verify_handler_func(self, handler_func):
+        handler_signature = inspect.signature(handler_func)
+        bad_parameters = ', '.join(
+            '%s is %s' % (p, p.kind)
+            for p in handler_signature.parameters.values()
+            if p.kind in (p.POSITIONAL_ONLY, p.VAR_POSITIONAL))
```

Comment:
Why don't we like positional params?

Reply:
We don't like position **only** parameters. This is [a new feature in Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#positional-only-parameters) (though builtin functions always had it):
```python
def foo(a, b, /, c, d): ...
```
`a` and `b` - the parameters before the `/` - are positional only, so you can't call them by name. You can do `foo(1, 2, 3, 4)` or `foo(1, 2, c=3, d=4)` but not `foo(a=1, b=2, c=3, d=4)`.

Since signal handlers get all their arguments by name, we'll need to do extra work to support positional arguments - and the RoI for that is too low, so it's better to just disallow them.